### PR TITLE
Fix some actions for plugins

### PR DESCRIPTION
**Pull request by rctdude2**
If an action doesn't have an AcceptParameters method, the parameters passed in executeAction will be ignored and the action will fail.

I only fixed the ones I found that were trivial, there are still some actions that involve coordinates that don't have AcceptParameters that I didn't modify because I'm not sure how they work in plugins. Also, I'm guessing this might need a network version increment, but I didn't do that because I wanted to ask first.

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
You'll need to increment the plug-in api version but no need to touch network version. The only other thing to think about is the naming of the variables. Are these consistent with the rest of the plug-in api I've not checked myself

File: src/openrct2/actions/SignSetNameAction.cpp [hunk 1]
```diff
@@ -26,6 +26,12 @@ SignSetNameAction::SignSetNameAction(BannerIndex bannerIndex, const std::string&
 {
 }
 
+void SignSetNameAction::AcceptParameters(GameActionParameterVisitor& visitor)
+{
+    visitor.Visit("banner", _bannerIndex);
```
Comment (IntelOrca) [inline src/openrct2/actions/SignSetNameAction.cpp:31]:
```suggestion
    visitor.Visit("id", _bannerIndex);
```

File: src/openrct2/actions/SetParkEntranceFeeAction.cpp [hunk 2]
```diff
@@ -20,6 +20,11 @@ SetParkEntranceFeeAction::SetParkEntranceFeeAction(money16 fee)
 {
 }
 
+void SetParkEntranceFeeAction::AcceptParameters(GameActionParameterVisitor& visitor)
+{
+    visitor.Visit("fee", _fee);
```
Comment (IntelOrca) [inline src/openrct2/actions/SetParkEntranceFeeAction.cpp:25]:
```suggestion
    visitor.Visit("value", _fee);
```

File: src/openrct2/actions/ParkSetLoanAction.cpp [hunk 3]
```diff
@@ -22,6 +22,11 @@ ParkSetLoanAction::ParkSetLoanAction(money32 value)
 {
 }
 
+void ParkSetLoanAction::AcceptParameters(GameActionParameterVisitor& visitor)
+{
+    visitor.Visit("loanAmount", _value);
```
Comment (IntelOrca) [inline src/openrct2/actions/ParkSetLoanAction.cpp:27]:
```suggestion
    visitor.Visit("value", _value);
```

File: src/openrct2/actions/ParkMarketingAction.cpp [hunk 4]
```diff
@@ -28,6 +28,13 @@ ParkMarketingAction::ParkMarketingAction(int32_t type, int32_t item, int32_t num
 {
 }
 
+void ParkMarketingAction::AcceptParameters(GameActionParameterVisitor& visitor)
+{
+    visitor.Visit("type", _type);
+    visitor.Visit("item", _item);
+    visitor.Visit("length", _numWeeks);
```
Comment (IntelOrca) [inline src/openrct2/actions/ParkMarketingAction.cpp:35]:
```suggestion
    visitor.Visit("duration", _numWeeks);
```

**Review by IntelOrca — COMMENTED** [review]
I have made some naming suggestions which I think fit better or are more generic - intuitive amongst other actions.

**Review by IntelOrca — APPROVED** [review]

**Pull request merged**